### PR TITLE
Fix cdo #776

### PR DESCRIPTION
**Pull request by ocefpaf**
Ping @DamienIrving.  Can you test this once the binaries land on the channel?

### CODE REVIEW

**Comment by DamienIrving** [comment]
@ocefpaf No problem. I'll know when they've landed by looking [here](https://anaconda.org/IOOS/cdo/files), right?

**Comment by ocefpaf** [comment]
>@ocefpaf No problem.

Thanks! I don't own a Mac to test.

>  I'll know when they've landed by looking here, right?

Yep.  The OS X CI (Travis-CI) says everything is OK. So merging it now. It should take a few hours for them to show up in the channel.

**Comment by DamienIrving** [comment]
@ocefpaf As far as I can tell they still haven't landed on the channel. Is that normal?

File: cdo/meta.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ source:
     md5: cca30c3c79335ad734e1838806f7bfc2
 
 build:
-    number: 1
+    number: 0
```
Comment (ocefpaf) [inline cdo/meta.yaml]: > @ocefpaf As far as I can tell they still haven't landed on the channel. Is that normal?

@DamienIrving my mistake.  This should be `2` and not `0`. Fixing this....

**Comment by DamienIrving** [comment]
@ocefpaf Just tested on Mac - it's all working fine.

**Comment by ocefpaf** [comment]
Thanks for testing it.

**Pull request merged**